packages/devtools_app/lib/src/debugger/flutter/debugger_controller.dart

```diff
@@ -406,7 +509,11 @@ class DebuggerController extends DisposableController
       return ref.uri == isolate.rootLib.uri;
     }, orElse: () => null);
 
-    showScriptLocation(ScriptLocation(mainScriptRef));
+    // Pre-prime the script info.
+    // ignore: unawaited_futures
+    getScript(mainScriptRef).then((script) {
+      showScriptLocation(ScriptLocation(mainScriptRef));
+    });
```

Comment:
this code is repeated here and above. put into helper

Reply:
done (`_populateScriptAndShowLocation`)